### PR TITLE
Sbottom update, local, CMG

### PR DESCRIPTION
**Pull request by safarzad**
adding c-tagger disc to jetTypeSUSY,
adding double_e noniso paths to 2016 trigger 
removing jetId and noise cleaned paths 
updating treeproducer and cfg file for sbottom analysis

### CODE REVIEW

**Comment by safarzad** [comment]
Is there anything to be done from my side to merge this pull request?

**Comment by gpetruc** [comment]
Looks fine for me, maybe somebody else can confirm that the changes in the triggers file are fine? 
(e.g. @mariadalfonso or @mschoene )

**Comment by safarzad** [comment]
I am trigger contact and all online cleaning stuff has been done by me  so I think I might know a bit of what is happening in the menu or at least of what I am doing there

**Comment by gpetruc** [comment]
I didn't want to imply that you don't know what you're doing, it's just
that a second pair of eyes may spot typos in what you wrote, e.g. a letter
that is uppercase when it should be lowercase or an extra or missing
underscore in a name.

File: RootTools/python/samples/triggers_13TeV_DATA2016.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 triggers_mumu = triggers_mumu_iso
 
 triggers_ee = [ "HLT_Ele17_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v*", "HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v*" ] # 17/12 prescaled in column 0
+triggers_ee_noniso = ["HLT_DoubleEle33_CaloIdL_v*", "LT_DoubleEle37_Ele27_CaloIdL_GsfTrkIdVL_v*", "HLT_DoubleEle33_CaloIdL_GsfTrkIdVL_v" ]
```
Comment (mschoene) [inline RootTools/python/samples/triggers_13TeV_DATA2016.py]: I think "LT_DoubleEle37_Ele27_CaloIdL_GsfTrkIdVL_v_" should be  "HLT_DoubleEle37_Ele27_CaloIdL_GsfTrkIdVL_v_"

**Comment by safarzad** [comment]
I edited it, is that OK?

**Comment by gpetruc** [comment]
Yes, it's ok now.

**Pull request merged**